locales/en-US/editor.properties

```diff
@@ -155,6 +154,16 @@ IMAGE_FILTER_GLOWING_SUN=Glowing Sun
 IMAGE_SAVE_WITH_FILTERS=Apply
 IMAGE_RESET_FILTERS=Cancel
 
+# Font Viewer
+FONT_FILE=Font File
+USE_FONT_INSTRUCTIONS=To use this font, copy and paste this code into your CSS file.
+FONT_TYPE_NORMAL=Normal
+FONT_TYPE_BOLD=Bold
+FONT_TYPE_ITALIC=Italic
+FONT_BOLD_HTML_TAG=Put text inside <code>&lt;strong&gt;</code><code>&lt;/strong&gt;</code> tag.
+FONT_BOLD_CSS_RULE=Add a <code>font-weight: bold;</code> rule.
+FONT_ITALIC_HTML_TAG=Put text inside <code>&lt;em&gt;</code><code>&lt;/em&gt;</code> tag.
+FONT_ITALIC_CSS_RULE=Add a <code>font-style: italic;</code> rule.
```

Comment:
This seems right, we just need to add them in Thimble.  Get Gideon to OK it, though.